### ISSUE TITLE
Installation for Pyton 3.7.7, Mac Catalina not succesful 

### ISSUE THREAD

**Issue by SONO641** [ReactionMechanismGenerator/RMG-Py#2083]
### Bug Description
Installation was unsuccessful for Mac OS Catalina, Python 3.7.7. Make test returned error with fail (skip=37, errors=8). 

### How To Reproduce
I tried installation following binary Installation using anaconda. I saw bunches of warnings related to Cython and Numpy. I removed the rmg_env and tried installation from the source. This also produced similar warnings. Make test ran gave an error and make eg1 also struggled. I also reproduced an error reported by Mark about Matplotlib using the binary installation. 

### Expected Behavior
My older version with Python 2 worked well!

### Installation Information
Describe your installation method and system information.
 - Mac OS 10.15.7
 - Installation method: followed user guide, from source
 - RMG version information: 3.0.0-394-gf07039c3c
   - RMG-Py: must be the latest 
   - RMG-database: must be the latest 

### Additional Context
below is a warning after "conda env create -f environment.yml"

Warning: 2 possible package resolutions (only showing differing packages):
  - defaults/noarch::parso-0.7.0-py_0, defaults/osx-64::jedi-0.17.2-py37hecd8cb5_1
  - defaults/noarch::parso-0.8.1-pyhd3eb1b0_0, defaults/osx-64::jedi-0.17.0-py37done

Then, below are examples of warning after "make"
```
[ 1/46] Cythonizing rmgpy/chemkin.pyx
warning: rmgpy/quantity.pxd:33:6: cpdef variables will not be supported in Cython 3; currently they are no different from cdef variables

/Users/shuheiono/anaconda3/envs/rmg_env/lib/python3.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:15:2: warning: "Using deprecated NumPy API, disable it by "
      "#defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-W#warnings]
#warning "Using deprecated NumPy API, disable it by " \

build/rmgpy/solver/base.c:12562:5: warning: code will never be executed [-Wunreachable-code]
    goto __pyx_L6_break;
    ^~~~~~~~~~~~~~~~~~~
```
These are errors after "make test" 
the first error is 
```
[20:08:11] WARNING: Proton(s) added/removed
ERROR:root:Unable to generate identifier for this molecule:
1 O u0 p3 c-1 {3,S}
2 O u0 p2 c0 {3,D}
3 N u0 p1 c0 {1,S} {2,D}
```
then, 
```
==============================
*** Open Babel Warning  in InChI code
  #0 :Accepted unusual valence(s): C(1)
==============================
*** Open Babel Warning  in InChI code
  #0 :Accepted unusual valence(s): C(1)
```
then, 
```
Traceback (most recent call last):
  File "rmgpy/reaction.py", line 546, in rmgpy.reaction.Reaction.get_free_energy_of_reaction
    dGrxn -= reactant.get_free_energy(T)
  File "rmgpy/species.py", line 527, in rmgpy.species.Species.get_free_energy
    raise Exception('Unable to calculate free energy for species {0!r}: '
Exception: Unable to calculate free energy for species 'C6H6-5': no thermo or statmech data available.
```
My own input file yielded a number of errors about no thermo etc. 

Many thanks for any help you could provide!

**Comment by amarkpayne** [comment]
@SONO641 can you upload a file with all of this? A lot of the warnings you are seeing are normal (the numpy cython warning aren't actually errors), so I am not seeing the cause of the problem yet with the messages you have posted. To help us out can you do the following?

1. Open a terminal, change your directory to your RMG-Py directory, and activate your rmg_env
2. Execute `make clean`
3. Execute `make >> python_installation_errors.txt`
4. Execute `make test >> python_installation_errors.txt`
5. Upload the python_installation_errors.txt document here from your RMG-Py folder.

**Comment by rwest** [comment]
Would appending ` 2>&1` to each of those commands be needed to capture the errors? (Perhaps not, but thought I'd ask)

**Comment by SONO641** [comment]
Many thanks for your reply!  I have just tried and am attaching error files below. 

[python_installation_errors.txt](https://github.com/ReactionMechanismGenerator/RMG-Py/files/6104596/python_installation_errors.txt)
[python_test_errors.txt](https://github.com/ReactionMechanismGenerator/RMG-Py/files/6104598/python_test_errors.txt)

**Comment by ChrisBNEU** [comment]
I am using catalina 10.15.7, I ran a fresh install and got the following. As a sidenote, the make test >> python_installation_errors.txt command doesn't seem to print out the whole log for the unit tests, so I just saved my terminal output. 
[Make_test_errors.txt](https://github.com/ReactionMechanismGenerator/RMG-Py/files/6318544/Make_test_errors.txt)
[python_installation_errors.txt](https://github.com/ReactionMechanismGenerator/RMG-Py/files/6318545/python_installation_errors.txt)

It looks like you can compile successfully so I don't think there are any problems with that. The unit test I believe has the same errors you posted(again, the make test output seems to be truncated). The errors you are listing are not failing any tests. Can you try running make test again, and saving your terminal output (go to shell > export text as)? I think we are still missing information.

**Comment by SONO641** [comment]
Hi, 
This is my terminal output from installation and make test.  Many thanks for looking into this! I hope this works. 

[terminalSavedOutput.txt](https://github.com/ReactionMechanismGenerator/RMG-Py/files/6324765/terminalSavedOutput.txt)

**Comment by ChrisBNEU** [comment]
I believe this actually was just fixed in #2091. What was failing was the Mopac unit tests because of a prior issue (#1864). if you do "git pull" from your master branch of rmg-py and run "make clean", then "make", it should work if you run the unit tests again. You should see something like the following at the end: 
```
------------------------------------------------------------------------
TOTAL                                  26299  12325  13482   1323    50%
----------------------------------------------------------------------
Ran 1806 tests in 227.026s

OK (SKIP=37)
```
